### ISSUE TITLE
Content for the "Web" use case

### ISSUE THREAD

**Issue by jasononeil** [HaxeFoundation/haxe.org#3]
If you have any content for the "web" use case page, please leave a comment here.

We are looking for:
- **Testimonials / Quotes**:  If you have used haxe for creating a website or web-app and it has been awesome, leave a short 1 sentance quote, with your name, company and URL.
- **Showcase**: If you have a website you would like included in the showcase, please include the name of your game, an image for the carousel, and a 1 line description.
- **Popular Libraries**: If there's a library that is used by a large number of Haxe web developers and we've left it out, let us know.
- **Tutorials**: if you have a tutorial, please supply a link, a title, and a 1 line description.

Please note we are not going to guarantee that items posted here will be included.  

Once we have enough content we can close this issue.

**Comment by Faba-network** [comment]
**Showcase**: http://shirtshop.landwirtschaft.ch/ Swiss "HTML5" Webshop. Frontend (js), backend (nodejs) and admin (js) build with Haxe.

**Comment by damoebius** [comment]
Showcase: http://heidi-module-personnalisation.com/demo/8-coussin.html 
Heidi is a product personalization cloud service. Composed of two parts, a Saas backend and webapp, the webapp, plus mobile components are powered by Haxe with the Saas backend written in C#.
preview : http://happy-technologies.com/wp-content/uploads/2014/09/module_heidi.png

**Comment by metaperl** [comment]
The classhomie app [mentioned here](http://haxe.org/use-cases/web/) is not up. Also the showcase links mentioned here are not added.

**Comment by Merelleya** [comment]
@jasononeil would you mind removing the classhomie app?

**Comment by jasononeil** [comment]
@Merelleya not at all.  The project was renamed to [Today We Learned](https://todaywelearned.co/) but I think there are better examples of Haxe for web.

@joergwasmeier I've added the ShirtShop - thanks, looks great.

@damoebius I was histant to add Heidi, despite being a very impressive UI, for two reasons:

1. Heidi, took quite a few clicks of linking my instagram account etc before I could see the interesting parts of the UI.  Also a bit harder as the UI is not in English.  (Neither is the UI for ShirtShop, but I found that easier to click around and explore anyway).
2. There was some user interface quirks that made it hard to use - when dragging and dropping an image around, my mouse cursor disappeared (Firefox 36.0.1) and this made it quite difficult to use.

Other than that it is a nice UI, but these are my reasons for not including it at this point.

**Comment by T1mL3arn** [comment]
ShirtShop links to the Net Wars.

**Comment by Merelleya** [comment]
Is it just me or is the shirtshop not loading properly? I get a loading screen and then nothing.

**Comment by Simn** [comment]
It works for me, but I can see that it's not working on your machine. Both browsers are chrome and the JS console doesn't report anything.

**Comment by joergwasmeier** [comment]
Hi there,

tanks for the link but it links for me to http://www.netwars-project.com/webdoc

@Merelleya hmm that doesnt sounds good :( I have tested it much with all Browsers and many people and i wonder that it doesn´t work for you. I use in this Project the frist time Websockets / Socket.io i thing that was not the best idea :( Well in the next week we roll out an update which than use normal post messages so it should work evrywhere and than you can buy Smartphones covers as well. 

greetings and have a nice week.

**Comment by Simn** [comment]
@joergwasmeier: I've fixed the link in https://github.com/HaxeFoundation/haxe.org/commit/fe9f7face25dff9be5764c873975d0f50939108f. Deploy is gonna take a few minutes.

**Comment by damoebius** [comment]
@jasononeil What's wrong with your mouse cursor ? Dont you have a hand cursor when dragging picture ?
Sorry if there is no English version actually. Coming soon

**Comment by dmouton** [comment]
as you can see at wwx, Heidi works well. Could you tell me if there is something wrong ?

**Comment by joergwasmeier** [comment]
Hey there, we have a new Project called KOPILOT. Its a german school cloud software used in 15 schools (and still counting ;) ). It is a big project (100k Haxe code, which we developed for 2 years ) using a new haxe framework called Synapse which we will release hopefully end this year (need to port from haxe 2 to haxe 3). The app uses Javascript, NodeJS, Mongodb, LessCSS. Actually you cant see much on the domain www.kopilot.de but on my domain http://joergwasmeier.de/portfolio-type/kopilot you can see more anyway perhaps it helps to show people that you can use haxe for big projects.

greetz

**Comment by felipevex** [comment]
Hi! Last year I started a project for a client. To help me with this project, I create a lib to development of web applications. This lib was released last month as open source and I hope it be useful for more people: http://lib.haxe.org/p/priori/

Some screenshots of the project I'm developing with Priori lib:

![captura de tela 2016-06-02 as 05 22 09](https://cloud.githubusercontent.com/assets/1934064/15738687/c518db54-2882-11e6-848e-dc4e18b211a3.png)
![captura de tela 2016-06-02 as 05 23 51](https://cloud.githubusercontent.com/assets/1934064/15738688/c51d4c16-2882-11e6-9f6a-d8e22bb5428a.png)
